components/Code/style.scss

```diff
@@ -5,6 +5,8 @@
 /* stylelint-disable scss/selector-no-redundant-nesting-selector */
 /* stylelint-disable order/properties-alphabetical-order */
 @mixin gfm-code-base-styles($background: #f6f8fa, $background-dark: #242e34, $text: inherit) {
+  --font-size: 95%;
```

Comment:
```suggestion
  --font-size: 90%;
```

I know I gave the option of 95% but I was comparing the original size to 95%, the inline text looks good but didn't consider it in a table. It's a bit large in table but I think 90% works. apologies and thanks!!
<img width="822" alt="image" src="https://github.com/readmeio/markdown/assets/104395799/aea31377-02b9-453b-9289-66b9d98ce011">